src/Microsoft.AspNetCore.Razor.Language/Legacy/CSharpCodeWriter.cs

```diff
@@ -651,5 +656,46 @@ public void Dispose()
             {
             }
         }
+
+        public struct StringLiteralWriter : IDisposable
```

Comment:
Proposition:
- Allows to keep the C# string syntax in CSharpCodeWriter.
- Reusable with the "IDisposable" pattern in the future.